SU-HSDO/suhumsci
# SHS-5645: Update multi-select box default text

# READY FOR REVIEW

## Summary
Update multi-select box default text

## Need Review By (Date)
06/19

## Urgency
medium

## Steps to Test

1. Go to the following pages
    - https://creativewriting.suhumsci.loc/books
    - https://politicalscience.suhumsci.loc/undergraduate-program/current-courses
    - https://hs-colorful.suhumsci.loc/default-views/courses
2. Confirm the multi-select box default text is `Choose one or more options`

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
